modules/setting/setting.go

```diff
@@ -157,6 +158,7 @@ var (
 		KeygenPath:          "ssh-keygen",
 		MinimumKeySizeCheck: true,
 		MinimumKeySizes:     map[string]int{"ed25519": 256, "ed25519-sk": 256, "ecdsa": 256, "ecdsa-sk": 256, "rsa": 2048},
+		ServerHostKeys:      []string{"ssh/gogs.rsa"},
```

Comment:
Maybe we could finally change it to gitea.rsa finally (I know it's breaking chage)?

Reply:
I think we can check if exist and move `ssh/gogs.rsa` to `ssh/gitea.rsa`

Reply:
hmm... I have an idea - if I make it `ServerHostKeys: []string{"ssh/gitea.rsa", "ssh/gogs.rsa"}` then it will use `gogs.rsa` if it exists but use create gitea.rsa if neither exists!